src/core/Akka/Configuration/Config.cs

```diff
@@ -6,11 +6,6 @@ namespace Akka.Configuration
 {
     public class Config
     {
-        private Config _fallback;
-        private HoconValue _node;
-        private IEnumerable<HoconSubstitution> _substitutions;
```

Comment:
removed private fields in favor for autoprops with private setters to support serialization.